writersolo.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/bash
+
+
+
+zypak-wrapper '/app/extra/opt/WriterSolo/${productName}-${version}-${arch}.${ext}'
```

Comment:
I don't see where productName version, arch and ext are defined.

Reply:
That's literally what the output binary of the executable is. It's literally named that. Not even installing the deb changed it. That's why this has been frustrating to troubleshoot.

Reply:
Add something in apply_extra no name it correctly